website/content/en/docs/best-practices/best-practices.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Operator Best Practices"
+linkTitle: "Best practices"
+weight: 1
+description: This guide describes the best practices concepts to write operators.
+---
+
+## Development
+
+Considerations for operator developers:
+
+- An Operator should manage a single type of application, essentially following the UNIX principle: do one thing and do it well.
+
+- If an application consists of multiple different tiers or components, multiple Operators should be written for each of them. If the application for example consists of Redis, AMQ and MySQL, there should be 3 Operators, not one.
+
+- If there is significant orchestration and sequencing involved, an Operator should be written that represents the entire stack, in turn delegating to other Operators for orchestrating their part of it.
+
+- Operators should own a CRD and only one Operator should control a CRD on a cluster. Two Operators managing the same CRD is not a recommended best practice. In the case where an API exists but with multiple implementations, this is typically an example of a no-op Operator because it doesn't have any deployment or reconciliation loop to define the shared API and other Operators depend on this Operator to provide one implementation of the API, e.g. similar to PVCs or Ingress.
+
+- Inside an Operator, multiple controllers should be used if multiple CRDs are managed. This helps with separation of concerns and code readability. Note that this doesn't necessarily mean one container image per controller, but rather one reconciliation loop (which could be running as part of the same operator binary) per CRD.
+
+- An operator shouldn't deploy or manage other operators (such patterns are known as meta or super operators). It's the Operator Lifecycle Manager's job to manage the deployment and lifecycle of operators.
```

Comment:
We probably need a document that describes how OLM solves this problem for these meta operators. Folks are creating these meta operators for a reason.

Reply:
Also, we should either always capitalize Operator or not, seems we do a mixture in this document. Let's pick one and use it consistently, I prefer "Operator" :)

Reply:
> We probably need a document that describes how OLM solves this problem for these meta operators. Folks are creating these meta operators for a reason.

That is the OLM Dependency. I think we can link that here.

Reply:
Done.